gtk/configure.ac

```diff
@@ -109,7 +109,7 @@ AC_SUBST(GLOBALLDFLAGS, $LDFLAGS )
 
 
 dnl ***************************************************************************
-dnl Internatinalization
+dnl Internationalization
```

Comment:
I actually excluded configure files in my final check, but this was misspelled and therefore I've left the correction.